src/editor/adapters/vscode-editor.ts

```diff
@@ -182,6 +193,83 @@ export class VSCodeEditor implements Editor {
     this.editor.selection = toVSCodeCursor(position);
     return Promise.resolve();
   }
+
+  async getSelectionReferences(selection: Selection): Promise<CodeReference[]> {
+    const locations = (await vscode.commands.executeCommand(
+      "vscode.executeReferenceProvider",
+      this.document.uri,
+      selection.start
+    )) as vscode.Location[];
+
+    const references = locations.map((loc) => {
```

Comment:
🍰 We may inline this temp variable and directly return the `locations.map()`

Reply:
Fixed :)